tests/e2e/monitoring/test_when_one_of_the_prometheus_node_down.py

```diff
@@ -0,0 +1,182 @@
+        self.namespace = create_unique_resource_name('test', 'namespace')
+        self.project_obj = ocp.OCP(kind='Project', namespace=self.namespace)
+        assert self.project_obj.new_project(self.namespace), (
+            f'Failed to create new project {self.namespace}'
+        )
+        # Create PVCs
+        self.pvc_obj = create_pvc(
+            sc_name=self.sc_obj.name, namespace=self.namespace
+        )
+
+        # Create pod
+        self.pod_obj = create_pod(
+            interface_type=constants.CEPHBLOCKPOOL,
+            pvc_name=self.pvc_obj.name, namespace=self.namespace
+        )
+
+        self.namespace_list.append(self.namespace)
+        self.pvc_objs.append(self.pvc_obj)
+        self.pod_objs.append(self.pod_obj)
+
+    # Check for the created pvc metrics is collected
+    for pvc_obj in self.pvc_objs:
+        assert collected_metrics_for_created_pvc(pvc_obj.name), (
+            f"On prometheus pod for created pvc {pvc_obj.name} related data is not collected"
+        )
+    return True
+
+
+def get_the_collected_metrics_for_pvcs_when_node_down():
+    """
+    Returns false if the metric/data are present
+    """
+    pvcs_data = get_kube_pod_spec_volumes_persistentvolumeclaims_info_metric()
+    pvcs_list = pvcs_data['data']['result']
+    if not pvcs_list:
+        logger.info("When one of the node down where prometheus"
+                    " hosted shouldn't be able to get the data/metrics")
+        return True
+    return False
+
+
+@pytest.mark.usefixtures(
+    create_rbd_secret.__name__,
+    create_ceph_block_pool.__name__,
+    create_rbd_storageclass.__name__,
+    test_fixture.__name__
+)
+@pytest.mark.polarion_id("OCS-606")
+class TestWhenOneOfThePrometheusNodeDown(E2ETest):
+    """
+    When the nodes are down, there should not be any functional impact
+    on monitoring pods. All the data/metrics should be collected correctly.
+    """
+    @tier4
+    def test_when_one_of_the_prometheus_node_down(self):
+        """
+        Test case to validate when the prometheus pod is down and
+        interaction with prometheus
+        """
+
+        # Get the pod obj for of the prometheus pod, i.e prometheusk8s-0
+        pod_obj = pod.get_pod_obj(name='prometheus-k8s-0', namespace=defaults.OCS_MONITORING_NAMESPACE)
+
+        # Get the node where the prometheus pod is hosted
+        prometheus_pod_obj = pod_obj.get()
+        prometheus_node = prometheus_pod_obj['spec']['nodeName']
+
+        # Get the node information
+        nodes_obj = ocp.OCP(kind='node')
+        nodes_list = nodes_obj.get()['items']
+        instances = [node for node in nodes_list if node['metadata']['name'] == prometheus_node]
+
+        # Make one of the node down where the prometheus pod is hosted
+        aws_obj = aws.AWS()
+        instance_dict = aws.get_instances_ids_and_names(instances)
+        aws_obj.stop_ec2_instances(instances=instance_dict, wait=True)
+
+        # Check for the created pvc metrics
```

Comment:
Can you also add validation to see if a new metric is generated when the node is down?

Reply:
will add in another PR.